styles/style.css

```diff
@@ -0,0 +1,542 @@
+  background-color: #3a3a3a !important;
+  border: 1px solid #555 !important;
+  color: #f1f1f1 !important;
+}
+
+.bitcoin-donation-message {
+  width: 100%;
+  resize: none;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  padding: 8px;
+  box-sizing: border-box;
+}
+
+.bitcoin-donation-title-wrapper {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+}
+
+.bitcoin-donation-title-wrapper h3 {
+  font-weight: bold;
+  /* color: #f7a70a; */
+}
+
+.bitcoin-donation-dark-theme h3 {
+  color: white !important;
+}
+
+
+
+/* Multi amount */
+
+.amount-wrapper {
+  position: relative;
+
+}
+
+.multi-form {
+  max-width: 280px;
+  padding: 32px 34px 42px 34px;
+  display: flex;
+  flex-direction: column;
+}
+
+.secondary-amount {
+  position: absolute;
+  top: 10px;
+  right: 9px;
+  font-size: 11px;
+  color: #9d9d9d;
+  user-select: none;
+  z-index: 999;
+}
+
+.snap-button {
+  width: 31% !important;
+  margin-top: 8px !important;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.snap-container {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+}
+
+.snap-title-container {
+  display: flex;
+  justify-content: center;
+}
+
+.snap-title-container h4 {
+  font-weight: bold;
+  margin-bottom: 4px;
+  margin-top: 8px;
+
+}
+
+.snap-secondary-amount {
+  font-size: 10px;
+  font-weight: 300;
+}
+
+.snap-primary-amount {
+  font-size: 12px;
+}
+
+.currency-swapper {
+  padding: 4px 8px !important;
+  border: 1px solid #e2e8f0 !important;
+  border-radius: 6px !important;
+  background-color: #f8f8f8 !important;
+  font-size: 12px !important;
+  font-weight: 500;
+  color: #1a202c;
+  cursor: pointer;
+  width: 31%;
+  text-align: center;
+  margin-bottom: 16px;
+}
+
+.bitcoin-donation-message {
+  min-height: 30px !important;
+}
+
+.currency-swapper:hover {
+  border-color: #cbd5e0;
+  background-color: #f7fafc;
+}
+
+.currency-swapper:focus {
+  outline: none;
+  border-color: #4299e1;
+  box-shadow: 0 0 0 3px rgba(66, 153, 225, 0.2);
+}
+
+.bitcoin-donation-dark-theme .currency-swapper {
+  background-color: #3a3a3a !important;
+  border: 1px solid #555 !important;
+  color: #f1f1f1 !important;
+}
+
+.bitcoin-donation-dark-theme .currency-swapper:hover {
+  background-color: #4a4a4a !important;
+  border: 1px solid #555 !important;
+  color: #f1f1f1 !important;
+}
+
+/* Voting */
+
+.poll-options {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+}
+
+.poll-option {
+  padding: 10px;
+  cursor: pointer;
+  background: #f2f2f2;
+  color: #3d3d3d;
+  /* border: 2px solid #6d6d6d; */
+  outline: 1px solid rgba(61, 61, 61, 0.5);
+  /* border: 1px solid #3d3d3d; */
+  border: none !important;
+  text-align: left;
+  border-radius: 5px;
+  transition: background 0.3s;
+}
+
+.poll-option:hover {
+  background: #d7d7d7;
+}
+
+.poll-results {
+  width: 100%;
+}
+
+.poll-result {
+  margin-bottom: 10px;
+}
+
+.progress {
```

Comment:
We should make these styles more specific, as there are a lot of themes and plugins there that use the `progress` class.